aggregator/log_manager.py

```diff
@@ -53,7 +53,7 @@ def get_log_entries(self, start_timestamp = 0, end_timestamp = None):
     if end_timestamp is None:
       end_timestamp = int(time.time())
 
-    self.cursor.execute('SELECT * FROM Log WHERE request_type = \'READ\' AND timestamp >= ? AND timestamp <= ?', (start_timestamp, int(end_timestamp)))
+    self.cursor.execute('SELECT * FROM Log WHERE request_type = \'READ\' AND status = 200 AND timestamp >= ? AND timestamp <= ?', (start_timestamp, int(end_timestamp)))
```

Comment:
hmm. i think transfer and replicate will still be aggregated into the log mgr. perhaps we need to write another method for retrieving transfer/replicate methods